docs/resources/azure_dns_zones_resource.md

```diff
@@ -0,0 +1,103 @@
+For an example `inspec.yml` file and how to set up your Azure credentials,
+refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+`resource_group` and dns_zones resource `name` or the `resource_id` must be given as a parameter.
+  ```ruby
+  describe azure_dns_zones_resource(resource_group: 'MyResourceGroup', name: 'dns_zones') do
+    it { should exist }
+  end
+  ```
+
+## Parameters
+
+| Name                           | Description                                                                      |
+|--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in. `MyResourceGroup`     |
+| name                           | Name of the Dns zones to test. `dns_name`                                 |
+
+
+## Properties
+
+| Name                           | Description                                                                      |
+  |--------------------------------|----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in. `MyResourceGroup`    |
+| name                           | Name of the Azure resource to test. `Mydns_zones`                          |
+| type                           | type of dns_zones                                                          |
+|max_number_of_recordsets | The maximum number of record sets that can be created in this DNS zone. This is a read-only property and any attempt to set this value will be ignored.|
+|number_of_record_sets| The current number of record sets in this DNS zone. This is a read-only property and any attempt to set this value will be ignored.|
+|name_servers|The name servers for this DNS zone. This is a read-only property and any attempt to set this value will be ignored.|
+| properties | properties of Azure resource group | `properties`|
+
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/dns/zones/get)
+for other properties available.
+Any attribute in the response may be accessed with the key names separated by dots (`.`).
+
+
+## Examples
+
+### Ensure that the dns_zones resource has is from same type
+  ```ruby
+  describe azure_dns_zones_resource(resource_group: 'MyResourceGroup', name: 'dns_zones') do
+    its('type') { should eq 'Microsoft.Network/dnszones' }
+  end
+  ```
+### Ensure that the dns_zones resource is in successful state
+  ```ruby
+  describe azure_dns_zones_resource(resource_group: 'MyResourceGroup', name: 'dns_zones') do
+    its('provisioning_state') { should include('Succeeded') }
+  end
+  ```
+
+### Ensure that the dns_zones resource is from same location
+  ```ruby
+  describe azure_dns_zones_resource(resource_group: 'MyResourceGroup', name: 'dns_zones') do
+    its('location') { should include df_location }
```

Comment:
Should `location` be in here?

Reply:
Yes Ian! It should be there .I missed in properties section .Added there too.